dump/sbe_dump_collector.cpp

```diff
@@ -246,16 +248,22 @@ bool SbeDumpCollector::logErrorAndCreatePEL(
                            "POSITION", chipPos);
             }
             // Processor FFDC Packets
-            openpower::dump::pel::processFFDCPackets(sbeError, event,
-                                                     pelAdditionalData);
+            logId = openpower::dump::pel::processFFDCPackets(sbeError, event,
+                                                             pelAdditionalData);
+            auto logInfo = openpower::dump::pel::getLogInfo(logId);
```

Comment:
here the exception from getLogInfo unhandled right? even if the getLogInfo failed we should continue, just need to skip addLogData to Dump